### ISSUE TITLE
Question: YOLOv26 parameter count mismatch vs docs

### ISSUE THREAD

**Issue by Farenweh** [ultralytics/ultralytics#23593]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Hi Ultralytics team,

I noticed a small mismatch between the parameter counts I get when inspecting the YOLOv26 models and the numbers listed in the documentation.

- yolo26n: measured 2.57M params vs docs 2.4M

- yolo26s: measured 10.1M params vs docs 9.5M

Could you clarify what accounts for this difference?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Farenweh, thank you for your interest in Ultralytics 🚀! This is an automated response to help us triage your issue quickly, and an Ultralytics engineer will also assist soon.

We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

To help us review your parameter-count mismatch report faster 🙏, please share:
-Your exact `ultralytics` version
-Your OS + Python + PyTorch versions
-The exact code/CLI you used to “inspect” params (and whether you loaded pretrained weights or built from YAML)
-Any relevant output logs or screenshots

Join the Ultralytics community where it suits you best 🙂. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by raimbekovm** [comment]
The docs are missing the one2one head params:

| Model | Actual | Docs (one2many only) |
|-------|--------|---------------------|
| yolo26n | 2.57M | 2.4M |
| yolo26s | 10.01M | 9.5M |

YOLO26 dual-head adds ~155K (n) / ~497K (s) extra params that weren't counted.

**Comment by glenn-jocher** [comment]
Confirmed — YOLO26 checkpoints ship with the dual-head (`one2many` + `one2one`/`end2end`) and `Model.info()` / `model_info()` counts **all** parameters in `model.parameters()`, so it will be higher than the docs table if the docs were computed from `one2many` only; we’ll sync the docs numbers accordingly (see the [`Model.info()` reference](https://docs.ultralytics.com/reference/engine/model/) and the [`Detect.one2many` / `Detect.one2one` properties](https://docs.ultralytics.com/reference/nn/modules/head/)).  

```python
from ultralytics import YOLO

y = YOLO("yolo26n.pt")
head = y.model.model[-1]  # Detect/Segment/Pose head

def nparams(mod):
    return sum(p.numel() for p in mod.parameters())

p_total = nparams(y.model)

p_one2many = sum(nparams(m) for m in head.one2many.values() if m is not None)
p_one2one  = sum(nparams(m) for m in head.one2one.values()  if m is not None)

print({"total": p_total, "one2many": p_one2many, "one2one": p_one2one})
```

**Comment by raimbekovm** [comment]
Nice — just to add: the docs numbers match the **fused** inference model (after `head.fuse()` strips the one2many branch + training-only modules).

This applies to both dual-head families — **YOLOv10** and **YOLO26**:

**YOLOv10 (detection)**

| Scale | Checkpoint | After `fuse()` | Docs |
|-------|:---:|:---:|:---:|
| n | 2.78M | 2.31M | 2.3M ✅ |
| s | 8.13M | 7.28M | 7.2M ✅ |
| m | 16.58M | 15.39M | 15.4M ✅ |
| l | 25.89M | 24.42M | 24.4M ✅ |
| x | 31.81M | 29.54M | 29.5M ✅ |

**YOLO26 (detection)**

| Scale | Checkpoint | After `fuse()` | Docs |
|-------|:---:|:---:|:---:|
| n | 2.57M | 2.42M | 2.4M ✅ |
| s | 10.01M | 9.51M | 9.5M ✅ |
| m | 21.90M | 20.44M | 20.4M ✅ |
| l | 26.30M | 24.84M | 24.8M ✅ |
| x | 58.99M | 55.77M | 55.7M ✅ |

Verified across all 20 YOLO26 task variants (seg, pose, obb, cls) — same pattern.
For seg/pose, `fuse()` also removes task-specific training modules (`Proto26.semseg`, `Pose26.flow_model`).

Happy to open a PR adding a footnote to the performance tables if that helps.

**Comment by glenn-jocher** [comment]
Good catch — agreed: the docs “Params” currently align with the post-`fuse()` inference graph (Conv+BN folding plus `Detect`/`Pose` dual-head pruning of `one2many`/training-only modules), whereas `Model.info()` on the raw checkpoint reports the full training graph, so it’ll be higher; a small docs-only PR adding a footnote clarifying “params after `fuse()`” would be welcome (see the [`Model.fuse()` reference](https://docs.ultralytics.com/reference/engine/model/) and [`BaseModel.fuse()` behavior](https://docs.ultralytics.com/reference/nn/tasks/)).  

```python
from ultralytics import YOLO

m = YOLO("yolo26n.pt")
m.fuse()
m.info()
```
